src/scenes/Partners/Edit/EditPartner.tsx

```diff
@@ -155,8 +163,16 @@ export const EditPartner = ({
       decorators={decorators}
       initialValues={initialValues}
       onSubmit={async ({
-        partner: { pointOfContactId, pmcEntityCode, ...rest },
+        partner: { pointOfContactId, pmcEntityCode, organizationName, ...rest },
       }) => {
+        if (organizationName && partner.organization.value) {
```

Comment:
The org name will always be truthy, even if it's not changed. You should check if this field is dirty instead. Here are some examples:
https://github.com/SeedCompany/cord-field/blob/f6001e70a1c304d64ef20ad30be88a7ae68645ec/src/scenes/Products/Edit/EditProduct.tsx#L259-L267
https://github.com/SeedCompany/cord-field/blob/f6001e70a1c304d64ef20ad30be88a7ae68645ec/src/scenes/Engagement/EditEngagement/EditEngagementDialog.tsx#L275-L281